client/src/index.ts

```diff
@@ -10,6 +10,7 @@ import Portal from './components/Portal/Portal';
 import PublicationStatus from './components/PublicationStatus/PublicationStatus';
 import Transition from './components/Transition/Transition';
 import { initUpgradeNotification } from './components/UpgradeNotification';
+export { initSkipLink } from './includes/initSkipLink';
```

Comment:
For consistency, could we update all the other imports to re-export like this?

Reply:
Good point.